projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -284,6 +296,28 @@ export class IgxInputGroupComponent extends DisplayDensityBase implements IgxInp
         return this.hints.length > 0;
     }
 
+    /** @hidden @internal */
+    @HostBinding('class.igx-input-group--prefixed')
+    public get hasPrefixes() {
```

Comment:
Better turn these into properties and not getters, since getters are treated as method calls which will cause `ExpressionChangedAfterItHasBeenCheckedError` as they may be invoked more than once in a change detection cycle. We should also make sure that `_prefixes` and `_suffixes` are defined, as well as `input` for the `isFileType` getter. Additionally, TS may complain about us using the `_prefixes` / `_suffixes` before they are initialized, in which case just wrap then in public getters and mark them as hidden/internal.